planner/core/task.go

```diff
@@ -413,6 +441,7 @@ func finishCopTask(ctx sessionctx.Context, task task) task {
 		p.stats = t.indexPlan.statsInfo()
 		newTask.p = p
 	} else {
+		splitCopAvg2CountAndSum(t.tablePlan)
```

Comment:
Should we do the split for `t.indexPlan` as well?

Reply:
I prefer not to do that. It's a requirement of the tiflash coprocessor. Although it doesn't regress the performance of the tikv coprocessor, it's not an improvement either. So I make its influence as little as possible.